src/virtual-code-owners/cst.d.ts

```diff
@@ -14,6 +20,18 @@ export interface IInterestingCSTLine {
   inlineComment: string;
   raw: string;
 }
+export interface ISectionCSTLine {
+  type: "section";
```

Comment:
technically a _section head_ (see the  GitLab documentation), the section is the  head + the rules. Maybe rename while we can?

Reply:
Good point. 

- [ ] Will update